engineering-feedback/readme.md

```diff
@@ -1,18 +1,20 @@
 # Microsoft Engineering Feedback
 
-## Why is it important to submit Microsoft Engineering Feedback?
+## Why is it important to submit Microsoft Engineering Feedback
 
-Engineering Feedback captures "voice of the customer" and is an important mechanism for helping Microsoft Product Group continuously improve the platform and cloud services and enable all customers to be as productive as possible.
+Engineering Feedback captures "voice of the customer" and is an important mechanism to provide actionable insights and help Microsoft Product Group continuously improve the platform and cloud services to enable all customers to be as productive as possible.

```

Comment:
I think this may be missing a "the" - "...insights and help **the** Microsoft Product Group ..."

Reply:
It was meant to be Microsoft Product Group**s**, fixed